### PR TITLE
Separate workflow for lint and test

### PR DESCRIPTION
**Pull request by Feiyang1**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,10 +31,10 @@
     "repl": "node tools/repl.js",
     "release": "ts-node-script scripts/release/cli.ts",
     "pretest": "node tools/pretest.js",
-    "test": "lerna run --concurrency 4 --stream test",
-    "test:ci": "lerna run --concurrency 4 --stream test:ci",
-    "test:release": "lerna run --concurrency 4 --ignore @firebase/*-exp --ignore firebase-exp --stream test:ci",
-    "test:exp": "lerna run --concurrency 4 --stream --scope @firebase/*-exp --scope firebase-exp test",
+    "test": "lerna run --concurrency 4 test",
```
Comment (hsubox76) [inline package.json:34]: Why turn off stream for regular (non-CI) tests? Seems helpful to see progress as it goes?

Comment (Feiyang1) [reply package.json]: Added it back.

File: package.json [hunk 2]
```diff
@@ -32,9 +32,9 @@
     "release": "ts-node-script scripts/release/cli.ts",
     "pretest": "node tools/pretest.js",
     "test": "lerna run --concurrency 4 --stream test",
-    "test:ci": "lerna run --concurrency 4 --stream test:ci",
-    "test:release": "lerna run --concurrency 4 --ignore @firebase/*-exp --ignore firebase-exp --stream test:ci",
-    "test:exp": "lerna run --concurrency 4 --stream --scope @firebase/*-exp --scope firebase-exp test",
+    "test:ci": "lerna run --concurrency 4 test:ci",
+    "test:release": "lerna run --concurrency 4 --ignore @firebase/*-exp --ignore firebase-exp test:ci",
+    "test:exp": "lerna run --concurrency 4 --scope @firebase/*-exp --scope firebase-exp test",
```
Comment (hsubox76) [inline package.json:37]: test:exp falls into the same category I think? It's not a CI command, right?

Comment (Feiyang1) [reply package.json]: yup, added it back

**Pull request merged**